ports/common/include/iot_network_mbedtls.h

```diff
@@ -51,17 +51,24 @@
 #define IOT_NETWORK_SERVER_INFO_MBEDTLS_INITIALIZER    { 0 }
 
 /**
- * @brief Initialize an #IotNetworkCredentials for AWS IoT with ALPN enabled
- * when using this mbed TLS network stack.
+ * @brief Initialize an #IotNetworkCredentials for AWS IoT for using TLS mutual
+ * authentication with certificates, TCP port 443, and mbedTLS.
  *
  * @note This initializer may change at any time in future versions, but its
  * name will remain the same.
  */
-#define AWS_IOT_NETWORK_CREDENTIALS_MBEDTLS_INITIALIZER \
-    {                                                   \
-        .pAlpnProtos = "x-amzn-mqtt-ca"                 \
+#define AWS_IOT_NETWORK_CREDENTIALS_MBEDTLS_INITIALIZER_FOR_CLIENT_CERTIFICATE \
```

Comment:
Can't change the name of this (per documentation note line 57)

Same goes for OpenSSL below.